roles/cis_security/defaults/main.yml

```diff
@@ -54,7 +54,7 @@ log_port: 514 # Linux: Port to listen to RSYSLOG messages on (if log_host is tru
 log_file_size: 8 # Linux: log file size. RHEL default is 8MB, control has no default
 # rsyslog_file:                  # Linux: Uncomment to copy file listed to /etc/rsyslog.d
 space_left_disk_size: 10000
-max_log_file_actions: keep_logs
+max_log_file_actions: rotate
```

Comment:
I need to reject this you should not change the default you should override it for our use case.

Reply:
got it reverting